src/MobileMapping/MobileMappingLayer.js

```diff
@@ -1,148 +0,0 @@
-/**
```

Comment:
You have totally removed MobileMappingLayer.js ?
If this is the case it is very good +1
It will come back at the right time